cfgov/unprocessed/css/on-demand/event.less

```diff
@@ -175,46 +174,92 @@
   vertical-align: middle;
 }
 
+.line-container {
```

Comment:
Carry over TODO:

```
// TODO: `line-container` is only used on events/_event-detail.html.
// Refactor to use DS classes.
```

Reply:
I don't think these TODOs are really necessary, as the entire presence of ANY code in `css/on-demand` (or the soon to be killed `css/pages` is always a tacit reminder that our patterns aren't being applied by default.

The first part of the comment is now unnecessary because the code isn't being built into `main.less` anymore--its presence as a one-off is enshrined.

Reply:
Fair enough. Maybe a comment at the top of all these classes that says they are used in `events/_event-detail.html`, since the file is imported into `event.html`, not `events/_event-detail.html`.

Reply:
Noted in 9db07eccb54f144a46cfabea3b62a0046b1731b8